apps/music/test/unit/player_view_test.js

```diff
@@ -0,0 +1,105 @@
+/* global PlayerView */
+
+'use strict';
+
+require('/js/ui/views/player_view.js');
+require('/shared/js/l10n.js');
```

Comment:
To deal with l10n you need to do the following:
* you need to include its mock instead:
```
require('/shared/test/unit/mocks/mock_l10n.js');
```

Then declare a variable around where pv and ratings are called 'realL10n'. Then, in suiteSetup replace the real l10n with the mock one (even if real one is undefined):
```
realL10n = navigator.mozL10n;
navigator.mozL10n = MockL10n;
```
Last thing is that you have to add suiteTeardown that will look like this (puts back the real l10n):
```
suiteTeardown(function() {
  navigator.mozL10n = realL10n;
});
```